### ISSUE TITLE
Medusa Develop SQLITE_ERROR

### ISSUE THREAD

**Issue by thoward27** [medusajs/medusa#1460]
Hello there! 

Apologies in advanced if I have missed something, but I believe there may be a bug in the current quickstart recommendations.

Here's what I did and the error I got back:

1. Install all dependencies listed here: https://docs.medusajs.com/tutorial/set-up-your-development-environment
2. `medusa new my-medusa-server --seed`
3. `cd my-medusa-server && medusa develop`
4. `curl localhost:9000/store/products | python -m json.tool`

The output I received from that is

```
{
    "code": "unknown_error",
    "message": "An unknown error occurred.",
    "type": "unknown_error"
}
```

Server error from that is 

```
error:   SQLITE_ERROR: no such column: price_list
QueryFailedError: SQLITE_ERROR: no such column: price_list
    at QueryFailedError.TypeORMError [as constructor] (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/TypeORMError.js:9:28)
    at new QueryFailedError (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/QueryFailedError.js:13:28)
    at handler (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:117:38)
    at replacement (/Users/thomashoward/Scratch/my-medusa-server/node_modules/sqlite3/lib/trace.js:25:27)
    at Statement.errBack (/Users/thomashoward/Scratch/my-medusa-server/node_modules/sqlite3/lib/sqlite3.js:14:21) {
  query: `SELECT "ma"."id" AS "ma_id", "ma"."currency_code" AS "ma_currency_code", "ma"."amount" AS "ma_amount", "ma"."min_quantity" AS "ma_min_quantity", "ma"."max_quantity" AS "ma_max_quantity", "ma"."price_list_id" AS "ma_price_list_id", "ma"."variant_id" AS "ma_variant_id", "ma"."region_id" AS "ma_region_id", "ma"."created_at" AS "ma_created_at", "ma"."updated_at" AS "ma_updated_at", "ma"."deleted_at" AS "ma_deleted_at", "price_list"."id" AS "price_list_id", "price_list"."name" AS "price_list_name", "price_list"."description" AS "price_list_description", "price_list"."type" AS "price_list_type", "price_list"."status" AS "price_list_status", "price_list"."starts_at" AS "price_list_starts_at", "price_list"."ends_at" AS "price_list_ends_at", "price_list"."created_at" AS "price_list_created_at", "price_list"."updated_at" AS "price_list_updated_at", "price_list"."deleted_at" AS "price_list_deleted_at" FROM "money_amount" "ma" LEFT JOIN "price_list" "price_list" ON "price_list"."id"="ma"."price_list_id" AND ( "ma"."price_list_id" = "price_list"."id"  AND "price_list"."deleted_at" IS NULL)  LEFT JOIN "price_list_customer_groups" "price_list_cgroup" ON "price_list_cgroup"."price_list_id"="price_list"."id" LEFT JOIN "customer_group" "cgroup" ON "cgroup"."id"="price_list_cgroup"."customer_group_id" AND ("cgroup"."deleted_at" IS NULL) WHERE ( "ma"."variant_id" = ? AND ("ma"."price_list_id" is null or "price_list"."status" = 'active') AND (price_list is null or "price_list"."ends_at" is null OR "price_list"."ends_at" > ?)  AND (price_list is null or "price_list"."starts_at" is null OR "price_list"."starts_at" < ?) AND "cgroup"."id" is null ) AND ( "ma"."deleted_at" IS NULL )`,
  parameters: [
    'variant_01G204QR6VSVX14KW62B0F86FC',
    'Sun, 01 May 2022 15:45:16 GMT',
    'Sun, 01 May 2022 15:45:16 GMT'
  ],
  driverError: Error: SQLITE_ERROR: no such column: price_list
  --> in Database#all(`SELECT "ma"."id" AS "ma_id", "ma"."currency_code" AS "ma_currency_code", "ma"."amount" AS "ma_amount", "ma"."min_quantity" AS "ma_min_quantity", "ma"."max_quantity" AS "ma_max_quantity", "ma"."price_list_id" AS "ma_price_list_id", "ma"."variant_id" AS "ma_variant_id", "ma"."region_id" AS "ma_region_id", "ma"."created_at" AS "ma_created_at", "ma"."updated_at" AS "ma_updated_at", "ma"."deleted_at" AS "ma_deleted_at", "price_list"."id" AS "price_list_id", "price_list"."name" AS "price_list_name", "price_list"."description" AS "price_list_description", "price_list"."type" AS "price_list_type", "price_list"."status" AS "price_list_status", "price_list"."starts_at" AS "price_list_starts_at", "price_list"."ends_at" AS "price_list_ends_at", "price_list"."created_at" AS "price_list_created_at", "price_list"."updated_at" AS "price_list_updated_at", "price_list"."deleted_at" AS "price_list_deleted_at" FROM "money_amount" "ma" LEFT JOIN "price_list" "price_list" ON "price_list"."id"="ma"."price_list_id" AND ( "ma"."price_list_id" = "price_list"."id"  AND "price_list"."deleted_at" IS NULL)  LEFT JOIN "price_list_customer_groups" "price_list_cgroup" ON "price_list_cgroup"."price_list_id"="price_list"."id" LEFT JOIN "customer_group" "cgroup" ON "cgroup"."id"="price_list_cgroup"."customer_group_id" AND ("cgroup"."deleted_at" IS NULL) WHERE ( "ma"."variant_id" = ? AND ("ma"."price_list_id" is null or "price_list"."status" = 'active') AND (price_list is null or "price_list"."ends_at" is null OR "price_list"."ends_at" > ?)  AND (price_list is null or "price_list"."starts_at" is null OR "price_list"."starts_at" < ?) AND "cgroup"."id" is null ) AND ( "ma"."deleted_at" IS NULL )`, [
    'variant_01G204QR6VSVX14KW62B0F86FC',
    'Sun, 01 May 2022 15:45:16 GMT',
    'Sun, 01 May 2022 15:45:16 GMT'
  ], [Function: handler])
      at SqliteQueryRunner.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:95:85)
      at step (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:144:27)
      at Object.next (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:125:57)
      at /Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:118:75
      at new Promise (<anonymous>)
      at __awaiter (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:114:16)
      at execute (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:86:78)
      at SqliteQueryRunner.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:139:46)
      at step (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:144:27)
      at Object.next (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:125:57)
      at fulfilled (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:115:62) {
    errno: 1,
    code: 'SQLITE_ERROR',
    __augmented: true
  },
  errno: 1,
  code: 'SQLITE_ERROR',
  __augmented: true
}
::1 - - [01/May/2022:15:45:16 +0000] "GET /store/products HTTP/1.1" 500 86 "-" "curl/7.77.0"
```

After hitting this, I did try to use `npx create-medusa-app`, but unfortunately, for me, that also fails, with this output

```
❯ npx create-medusa-app
✔ Where should your project be installed? · my-medusa-store
✔ Which Medusa starter would you like to install? · medusa-starter-contentful
✔ Which storefront starter would you like to install? · Next.js Starter
⠦ Creating new project from git: https://github.com/medusajs/medusa-starter-contentful.git
✔ Created starter directory layout

⠙ Installing packages...
✔ Packages installed
Initialising git in my-medusa-store/backend
Create initial git commit in my-medusa-store/backend
⠋ Seeding database
> medusa-starter-contentful@0.0.1 seed
> medusa seed -f ./data/seed.json

[medusa-config] ⚠️ jwt_secret not found. fallback to either cookie_secret or default 'supersecret'.
[medusa-config] ⚠️ cookie_secret not found. fallback to either cookie_secret or default 'supersecret'.
TypeError: Expected parameter accessToken
    at createCMAHttpClient (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7479:11)
    at createClient (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7424:99)
    at new ContentfulService (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/services/contentful.js:80:64)
    at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:377:112
    at Object.resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/resolvers.js:295:20)
    at resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:250:41)
    at Object.get (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:67:32)
    at new ContentfulSubscriber (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/subscribers/contentful.js:23:32)
    at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:77
    at Object.resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/resolvers.js:295:20)
    at Object.build (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:313:37)
    at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:19
    at Array.forEach (<anonymous>)
    at registerSubscribers (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:403:11)
    at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:92:41
    at step (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:33:23)
⠧ Seeding database
```

Unfortunately, this is where the process hangs.

**Comment by thoward27** [comment]
Additionally, this seems problematic

```
❯ npx create-medusa-app --no-seed
✔ Where should your project be installed? · taylor-made
✔ Which Medusa starter would you like to install? · medusa-starter-contentful
✔ Which storefront starter would you like to install? · Next.js Starter
⠋ Creating new project from git: https://github.com/medusajs/medusa-starter-contentful.git
✔ Created starter directory layout

⠧ Installing packages...
✔ Packages installed
Initialising git in taylor-made/backend
Create initial git commit in taylor-made/backend
⠋ Seeding database
> medusa-starter-contentful@0.0.1 seed
> medusa seed -f ./data/seed.json

[medusa-config] ⚠️ jwt_secret not found. fallback to either cookie_secret or default 'supersecret'.
[medusa-config] ⚠️ cookie_secret not found. fallback to either cookie_secret or default 'supersecret'.
TypeError: Expected parameter accessToken
    at createCMAHttpClient (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7479:11)
    at createClient (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7424:99)
    at new ContentfulService (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/medusa-plugin-contentful/services/contentful.js:80:64)
    at /Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:377:112
    at Object.resolve (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/awilix/lib/resolvers.js:295:20)
    at resolve (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/awilix/lib/container.js:250:41)
    at Object.get (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/awilix/lib/container.js:67:32)
    at new ContentfulSubscriber (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/medusa-plugin-contentful/subscribers/contentful.js:23:32)
    at /Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:77
    at Object.resolve (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/awilix/lib/resolvers.js:295:20)
    at Object.build (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/awilix/lib/container.js:313:37)
    at /Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:19
    at Array.forEach (<anonymous>)
    at registerSubscribers (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:403:11)
    at /Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:92:41
    at step (/Users/thomashoward/Scratch/taylor-made/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:33:23)
⠴ Seeding database
```

I was hoping I could initialize the codebase without seeding the db, but that option seems broken

**Comment by olivermrbl** [comment]
This seems to be an issue with SQLite, will investigate. In the meantime, can I get you to try to set up Postgres for your server?

In `medusa-config.js` change the project configuration in the bottom of the file to something along the lines of:
```
module.exports = {
  projectConfig: {
    // redis_url: REDIS_URL,
    database_url: DATABASE_URL,
    database_type: "postgres",
    store_cors: STORE_CORS,
    admin_cors: ADMIN_CORS,
  },
  plugins,
};
```

And create a Postgres DB locally. Default name is `medusa-store`.

**Comment by thoward27** [comment]
@olivermrbl is there a recommended way to stand up the postgres database? I used `createdb medusa-store`, then made the requested changes to my config file, I get a bunch of errors like this one:

```
Initializing defaultsQueryFailedError: relation "staged_job" does not exist
    at QueryFailedError.TypeORMError [as constructor] (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/TypeORMError.js:9:28)
    at new QueryFailedError (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/QueryFailedError.js:13:28)
    at PostgresQueryRunner.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/postgres/PostgresQueryRunner.js:281:31)
    at step (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:144:27)
    at Object.throw (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:125:57)
    at rejected (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:116:69)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  query: 'SELECT "StagedJob"."id" AS "StagedJob_id", "StagedJob"."event_name" AS "StagedJob_event_name", "StagedJob"."data" AS "StagedJob_data" FROM "staged_job" "StagedJob"',
  parameters: [],
  driverError: error: relation "staged_job" does not exist
      at Parser.parseErrorMessage (/Users/thomashoward/Scratch/my-medusa-server/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/Users/thomashoward/Scratch/my-medusa-server/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/Users/thomashoward/Scratch/my-medusa-server/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:390:28)
      at addChunk (node:internal/streams/readable:315:12)
      at readableAddChunk (node:internal/streams/readable:289:9)
      at Socket.Readable.push (node:internal/streams/readable:228:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:199:23) {
```

I can paste the entire output if that would be helpful here.

Is there a way to instantiate the database so that `medusa develop` can bootstrap against postgres?

**Comment by olivermrbl** [comment]
Have you run `yarn seed` after having set up Postgres? @thoward27

**Comment by thoward27** [comment]
That's the bit I was missing @olivermrbl! After running `yarn seed` the BE appears to come up cleanly, curl does work

**Comment by olivermrbl** [comment]
Great! Will try to dig into the issue with SQLite. Seems the migrations are not applied.

**Comment by murtuzaalisurti** [comment]
> Hello there!
> 
> Apologies in advanced if I have missed something, but I believe there may be a bug in the current quickstart recommendations.
> 
> Here's what I did and the error I got back:
> 
> 1. Install all dependencies listed here: https://docs.medusajs.com/tutorial/set-up-your-development-environment
> 2. `medusa new my-medusa-server --seed`
> 3. `cd my-medusa-server && medusa develop`
> 4. `curl localhost:9000/store/products | python -m json.tool`
> 
> The output I received from that is
> 
> ```
> {
>     "code": "unknown_error",
>     "message": "An unknown error occurred.",
>     "type": "unknown_error"
> }
> ```
> 
> Server error from that is
> 
> ```
> error:   SQLITE_ERROR: no such column: price_list
> QueryFailedError: SQLITE_ERROR: no such column: price_list
>     at QueryFailedError.TypeORMError [as constructor] (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/TypeORMError.js:9:28)
>     at new QueryFailedError (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/error/QueryFailedError.js:13:28)
>     at handler (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:117:38)
>     at replacement (/Users/thomashoward/Scratch/my-medusa-server/node_modules/sqlite3/lib/trace.js:25:27)
>     at Statement.errBack (/Users/thomashoward/Scratch/my-medusa-server/node_modules/sqlite3/lib/sqlite3.js:14:21) {
>   query: `SELECT "ma"."id" AS "ma_id", "ma"."currency_code" AS "ma_currency_code", "ma"."amount" AS "ma_amount", "ma"."min_quantity" AS "ma_min_quantity", "ma"."max_quantity" AS "ma_max_quantity", "ma"."price_list_id" AS "ma_price_list_id", "ma"."variant_id" AS "ma_variant_id", "ma"."region_id" AS "ma_region_id", "ma"."created_at" AS "ma_created_at", "ma"."updated_at" AS "ma_updated_at", "ma"."deleted_at" AS "ma_deleted_at", "price_list"."id" AS "price_list_id", "price_list"."name" AS "price_list_name", "price_list"."description" AS "price_list_description", "price_list"."type" AS "price_list_type", "price_list"."status" AS "price_list_status", "price_list"."starts_at" AS "price_list_starts_at", "price_list"."ends_at" AS "price_list_ends_at", "price_list"."created_at" AS "price_list_created_at", "price_list"."updated_at" AS "price_list_updated_at", "price_list"."deleted_at" AS "price_list_deleted_at" FROM "money_amount" "ma" LEFT JOIN "price_list" "price_list" ON "price_list"."id"="ma"."price_list_id" AND ( "ma"."price_list_id" = "price_list"."id"  AND "price_list"."deleted_at" IS NULL)  LEFT JOIN "price_list_customer_groups" "price_list_cgroup" ON "price_list_cgroup"."price_list_id"="price_list"."id" LEFT JOIN "customer_group" "cgroup" ON "cgroup"."id"="price_list_cgroup"."customer_group_id" AND ("cgroup"."deleted_at" IS NULL) WHERE ( "ma"."variant_id" = ? AND ("ma"."price_list_id" is null or "price_list"."status" = 'active') AND (price_list is null or "price_list"."ends_at" is null OR "price_list"."ends_at" > ?)  AND (price_list is null or "price_list"."starts_at" is null OR "price_list"."starts_at" < ?) AND "cgroup"."id" is null ) AND ( "ma"."deleted_at" IS NULL )`,
>   parameters: [
>     'variant_01G204QR6VSVX14KW62B0F86FC',
>     'Sun, 01 May 2022 15:45:16 GMT',
>     'Sun, 01 May 2022 15:45:16 GMT'
>   ],
>   driverError: Error: SQLITE_ERROR: no such column: price_list
>   --> in Database#all(`SELECT "ma"."id" AS "ma_id", "ma"."currency_code" AS "ma_currency_code", "ma"."amount" AS "ma_amount", "ma"."min_quantity" AS "ma_min_quantity", "ma"."max_quantity" AS "ma_max_quantity", "ma"."price_list_id" AS "ma_price_list_id", "ma"."variant_id" AS "ma_variant_id", "ma"."region_id" AS "ma_region_id", "ma"."created_at" AS "ma_created_at", "ma"."updated_at" AS "ma_updated_at", "ma"."deleted_at" AS "ma_deleted_at", "price_list"."id" AS "price_list_id", "price_list"."name" AS "price_list_name", "price_list"."description" AS "price_list_description", "price_list"."type" AS "price_list_type", "price_list"."status" AS "price_list_status", "price_list"."starts_at" AS "price_list_starts_at", "price_list"."ends_at" AS "price_list_ends_at", "price_list"."created_at" AS "price_list_created_at", "price_list"."updated_at" AS "price_list_updated_at", "price_list"."deleted_at" AS "price_list_deleted_at" FROM "money_amount" "ma" LEFT JOIN "price_list" "price_list" ON "price_list"."id"="ma"."price_list_id" AND ( "ma"."price_list_id" = "price_list"."id"  AND "price_list"."deleted_at" IS NULL)  LEFT JOIN "price_list_customer_groups" "price_list_cgroup" ON "price_list_cgroup"."price_list_id"="price_list"."id" LEFT JOIN "customer_group" "cgroup" ON "cgroup"."id"="price_list_cgroup"."customer_group_id" AND ("cgroup"."deleted_at" IS NULL) WHERE ( "ma"."variant_id" = ? AND ("ma"."price_list_id" is null or "price_list"."status" = 'active') AND (price_list is null or "price_list"."ends_at" is null OR "price_list"."ends_at" > ?)  AND (price_list is null or "price_list"."starts_at" is null OR "price_list"."starts_at" < ?) AND "cgroup"."id" is null ) AND ( "ma"."deleted_at" IS NULL )`, [
>     'variant_01G204QR6VSVX14KW62B0F86FC',
>     'Sun, 01 May 2022 15:45:16 GMT',
>     'Sun, 01 May 2022 15:45:16 GMT'
>   ], [Function: handler])
>       at SqliteQueryRunner.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:95:85)
>       at step (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:144:27)
>       at Object.next (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:125:57)
>       at /Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:118:75
>       at new Promise (<anonymous>)
>       at __awaiter (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:114:16)
>       at execute (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:86:78)
>       at SqliteQueryRunner.<anonymous> (/Users/thomashoward/Scratch/my-medusa-server/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:139:46)
>       at step (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:144:27)
>       at Object.next (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:125:57)
>       at fulfilled (/Users/thomashoward/Scratch/my-medusa-server/node_modules/tslib/tslib.js:115:62) {
>     errno: 1,
>     code: 'SQLITE_ERROR',
>     __augmented: true
>   },
>   errno: 1,
>   code: 'SQLITE_ERROR',
>   __augmented: true
> }
> ::1 - - [01/May/2022:15:45:16 +0000] "GET /store/products HTTP/1.1" 500 86 "-" "curl/7.77.0"
> ```
> 
> After hitting this, I did try to use `npx create-medusa-app`, but unfortunately, for me, that also fails, with this output
> 
> ```
> ❯ npx create-medusa-app
> ✔ Where should your project be installed? · my-medusa-store
> ✔ Which Medusa starter would you like to install? · medusa-starter-contentful
> ✔ Which storefront starter would you like to install? · Next.js Starter
> ⠦ Creating new project from git: https://github.com/medusajs/medusa-starter-contentful.git
> ✔ Created starter directory layout
> 
> ⠙ Installing packages...
> ✔ Packages installed
> Initialising git in my-medusa-store/backend
> Create initial git commit in my-medusa-store/backend
> ⠋ Seeding database
> > medusa-starter-contentful@0.0.1 seed
> > medusa seed -f ./data/seed.json
> 
> [medusa-config] ⚠️ jwt_secret not found. fallback to either cookie_secret or default 'supersecret'.
> [medusa-config] ⚠️ cookie_secret not found. fallback to either cookie_secret or default 'supersecret'.
> TypeError: Expected parameter accessToken
>     at createCMAHttpClient (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7479:11)
>     at createClient (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/node_modules/contentful-management/dist/contentful-management.node.js:7424:99)
>     at new ContentfulService (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/services/contentful.js:80:64)
>     at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:377:112
>     at Object.resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/resolvers.js:295:20)
>     at resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:250:41)
>     at Object.get (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:67:32)
>     at new ContentfulSubscriber (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/medusa-plugin-contentful/subscribers/contentful.js:23:32)
>     at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:77
>     at Object.resolve (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/resolvers.js:295:20)
>     at Object.build (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/awilix/lib/container.js:313:37)
>     at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:405:19
>     at Array.forEach (<anonymous>)
>     at registerSubscribers (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:403:11)
>     at /Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:92:41
>     at step (/Users/thomashoward/Scratch/my-medusa-store/backend/node_modules/@medusajs/medusa/dist/loaders/plugins.js:33:23)
> ⠧ Seeding database
> ```
> 
> Unfortunately, this is where the process hangs.

Facing the same issue!

**Comment by murtuzaalisurti** [comment]
> Have you run `yarn seed` after having set up Postgres? @thoward27

I followed the steps but then I am getting this!

![Code_Z3P4XRwDbm](https://user-images.githubusercontent.com/68743212/166290891-65ac406c-de36-4555-a430-cd894d93b1e8.jpg)

**Comment by adrien2p** [comment]
Hey @murtuzaalisurti @thoward27 @olivermrbl , It should be fixed by [this PR](https://github.com/medusajs/medusa/pull/1462)

**Comment by murtuzaalisurti** [comment]
> Have you run `yarn seed` after having set up Postgres? @thoward27

After solving lot of postgres errors, this worked for me. I am using postgres and redis in development. Unfortunately, sqlite didn't work for me!

**Comment by adrien2p** [comment]
@murtuzaalisurti the sqlite error will be fixed with the pr I mentioned above 🤘

**Comment by murtuzaalisurti** [comment]
> @murtuzaalisurti the sqlite error will be fixed with the pr I mentioned above 🤘

That's great

**Comment by olivermrbl** [comment]
Fixed #1462
